api-contract/src/main/java/uk/gov/hmcts/payment/api/contract/CreditAccountPaymentRequest.java

```diff
@@ -38,6 +38,8 @@
     @NotEmpty
     private String description;
 
+    @NotNull
```

Comment:
Just to make sure is services aware of this changes and i hope there is no impact and tested all the services who are on-boarded to PBA with both manual and functional tests?